### ISSUE TITLE
Support for modules/import

### ISSUE THREAD

**Issue by pke** [laggingreflex/mochista#18]
It seems mochista is not yet ready for modules?
Using `import` is not possible it seems.

**Comment by laggingreflex** [comment]
Yes, that is the case I'm afraid. Since none of the underlying libraries support it (mocha, c8) it doesn't either. 

I'm not sure if it works, but you can try https://github.com/standard-things/esm. (I remember running into issues a long time ago when I tried it, maybe it works now)

You can also try running node with the appropriate flags needed to enable ESM module support natively. If node natively supports it, it should work.

**Comment by pke** [comment]
So that means upgrading node to the newest version to get import support. I'll try.
